src/main/java/tb/common/block/BlockTBPlant.java

```diff
@@ -153,6 +153,10 @@ public void onBlockPlacedBy(World worldIn, int x, int y, int z, EntityLivingBase
     @Override
     public boolean onBlockActivated(World aWorld, int aX, int aY, int aZ, EntityPlayer aPlayer, int aSide, float pX,
         float pY, float pZ) {
+        if (aWorld.getBlock(aX, aY, aZ) instanceof BlockAshroom) {
```

Comment:
This feels a bit dirty, why didn't you just override the method in `BlockAshroom`?

Reply:
Simply didnt know better